packages/self_storing_input/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.1
```

Comment:
This seems wrong? You are simply renaming.

Reply:
I am renaming and adding body. Line 48.